### DISCUSSION TITLE
repository jekyll-generator-single-source not found

### DISCUSSION THREAD

**Discussion by zhongweikang** [Kong/kong#12856] [Help] [answered]
I am getting the following error while deploying the docs.konghq.com project:

```
$ make install
git submodule update --init
npm ci

added 1119 packages, and audited 1120 packages in 1m

262 packages are looking for funding
  run `npm fund` for details

12 vulnerabilities (2 moderate, 2 high, 8 critical)

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
bundle install
Fetching gem metadata from https://rubygems.org/.........
Fetching https://github.com/kong/jekyll-generator-single-source.git
remote: Repository not found.
fatal: repository 'https://github.com/kong/jekyll-generator-single-source.git/' not found
make: *** [install] Error 11
```

**Comment by fabianrbz** [comment]
@zhongweikang thanks for the issue report.
It looks like we were pointing the dependency `jekyll-generator-single-source` to a specific branch in https://github.com/kong/jekyll-generator-single-source (private repo) instead of rubygems (packet manager) when you tried to run `make install`.
We released a new version of the gem since then and no longer point to the branch. Can you try again?

**Reply by zhongweikang** [reply]
I've tried. It's still the same error.

**Reply by fabianrbz** [reply, marked as answer]
You probably need to fetch the latest changes first.
Try running `git pull origin main` first and then `make install`.

Line number 7 on your local version of the Gemfile should look like [this](https://github.com/Kong/docs.konghq.com/blob/main/Gemfile#L7).

**Reply by zhongweikang** [reply]
Sorry, I forgot to pull the code.

It worked, thanks for your help!
